code/framework/object/src/main/java/io/cattle/platform/object/meta/impl/DefaultObjectMetaDataManager.java

```diff
@@ -612,6 +612,10 @@ public SchemaImpl postProcess(SchemaImpl schema, SchemaFactory factory) {
         Map<String, Relationship> result = new HashMap<String, Relationship>();
         Schema schema = schemaFactory.getSchema(type);
 
+        if (null == schema) {
```

Comment:
This is quite scary changing a core class like this.  I really don't know the repercussions of this.